games-emulation/retroarch/retroarch-1.7.1.recipe

```diff
@@ -0,0 +1,89 @@
+SUMMARY="The reference frontend of libretro"
+DESCRIPTION="RetroArch is a frontend for emulators, game engines and media \
+players. It enables you to run classic games on a wide range of computers and \
+consoles through its slick graphical interface. It has advanced features like \
+shaders, netplay, rewinding, next-frame response times, and more!"
+HOMEPAGE="https://libretro.com/"
+COPYRIGHT="2010-2018 The RetroArch Team"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/libretro/RetroArch/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="e9189c7da6b2ca0142528eca14f26104ee322f052f8609203d90b60938d8d3f3"
+SOURCE_FILENAME="retroarch-$portVersion.tar.gz"
+SOURCE_DIR="RetroArch-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+GLOBAL_WRITABLE_FILES="
+	$relativeSettingsDir/retroarch.cfg auto-merge
+	"
+
+USER_SETTINGS_FILES="
+	$relativeSettingsDir/retroarch/retroarch.cfg template \
+		$sysconfDir/retroarch.cfg
```

Comment:
Hi @fbrosson 
I've been using `hp -v` since the very beginning, this alias is set as such 
```
hp='haikuporter -S -j4 --get-dependencies --no-source-packages'
```
With the current code `retroarch` builds and installs just fine :)

Reply:
:+1: for using `-S` ;)

Regarding the template, I'm a bit worried by the path we have after the `template` keyword. Here, we are passing the absolute path that is equivalent to the path of the config itself.
My understanding is that what we pass as template should be a relative path to a file in a read-only directory like `$dataDir/retroarch/retroarch.cfg`.

Reply:
@fbrosson They're not equivalent, see [Infrastructure](https://git.haiku-os.org/haiku/tree/docs/develop/packages/Infrastructure.rst#n171)